policyengine_us/parameters/gov/states/ct/tax/income/credits/property_tax/eligibility_available.yaml

```diff
@@ -0,0 +1,15 @@
+description: Connecticut utilizes the age of the filer as the eligibility for the property tax credit if this is true.
```

Comment:
```suggestion
description: Connecticut limits the property tax credit to filers above a certain age or having a dependent in the household if this is true, otherwise every filer is eligible.
```